### ISSUE TITLE
Error parsing `<script>` tags with a callback in Svelte

### ISSUE THREAD

**Issue by Aloso** [oxc-project/oxc#20349]
`<script>` tags containing a callback are incorrectly parsed in Svelte files.

### Minimal example

```svelte
{#if browser}
  <script src="/" onload={() => {}}></script>
{/if}
```

### Parse error reported by oxlint:

```
  × Unexpected token
   ╭─[src/lib/components/test.svelte:2:35]
 1 │ {#if browser}
 2 │   <script src="/" onload={() => {}}></script>
   ·                                   ─
 3 │ {/if}
   ╰────
```

Version: 0.40.0

**Comment by camc314** [comment]
is this oxfmt or oxlint?

**Comment by Aloso** [comment]
oxlint
